first Document.txt

```diff
@@ -0,0 +1,7 @@
+jdoidheiisdvblv
```

Comment:
What does his text mean?

Reply:
It means nothing, just jargon to learn commands